src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -80,18 +79,13 @@ private ChangeList ComputeChildren(TextDocument left, TextDocument right, Cancel
 
             diffService = diffService ?? diffSelector.DefaultTextDifferencingService;
 
-            var diff = ComputeDiffSpans(diffService, left, right, cancellationToken);
-            if (diff.Differences.Count == 0)
+            var diff = ComputeDiffSpans(oldText, newText, diffService, cancellationToken);
+            if (!diff.Differences.Any())
             {
-                // There are no changes.
-                return ChangeList.Empty;
+                // There are no changes. Just show the whole document.
+                return GetEntireDocumentAsSpanChange(left);
```

Comment:
?? why showing whole document? why moving this behavior back? does this has anything to do with changing diffing to Word?